backend/inspirehep/migrator/cli.py

```diff
@@ -156,3 +157,41 @@ def continuously():
             touch_file(liveness_file)
         continuous_migration()
         sleep(current_app.config.get("MIGRATION_POLLING_SLEEP", 1))
+
+
+@migrate.command()
+@click.option(
+    "-f",
+    "--force",
+    is_flag=True,
+    default=False,
+    help="Force the task to run even in debug mode.",
+)
+@click.option(
+    "-w",
+    "--wait",
+    is_flag=True,
+    default=False,
+    help="Wait for all subtasks to finish. (No progress bar will be available still)",
```

Comment:
```suggestion
    help="Wait for all subtasks to finish without progress bar.",
```

Reply:
It means that there is no progress bar with out without this option. Your help message suggests that without this option there is progress bar.

Reply:
I'm missing something :/ it's still not clear for me, this option will display a progressbar or not?

Reply:
This option has nothing to do with progressbar, This comment is there only because old migration had progressbar. Comment is copy-paste of same option from `mirror` command.

Reply:
let's remove then ``` (No progress bar will be available still)`` we don't need it right?

Reply:
:+1: